fuzzinator/executor.py

```diff
@@ -66,6 +66,9 @@ def process_args(args):
     args.config = config
 
     inators.arg.process_log_level_argument(args, root_logger)
+    # Force chatty third-party libraries to report at least on INFO level.
+    for logger_name in ['asyncio', 'chardet.charsetprober', 'keyring.backend']:
+        logging.getLogger(logger_name).setLevel(max([logging.getLevelName(args.log_level), logging.INFO]))
```

Comment:
If I'm not mistaken, this will not work with "TRACE" or "DISABLE" log levels (added by the inators package, and also handled by `inators.arg.process_log_level_argument` above). 

```py
>>> import logging
>>> arr = [logging.getLevelName("TRACE"), logging.INFO]
>>> arr
['Level TRACE', 20]
>>> max(arr)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'int' and 'str'
```

Also, the max needs to be computed only once. And a tuple is enough for the for loop, an array is not necessary.

So, something like the snippet below might work a bit better:

```py
# Force chatty third-party libraries to report at least on INFO level.
log_level_chatty = max(inators.log.levels[args.log_level], inators.log.INFO)
for logger_name in ('asyncio', 'chardet.charsetprober', 'keyring.backend'):
    logging.getLogger(logger_name).setLevel(log_level_chatty)
```